index.d.ts

```diff
@@ -0,0 +1,159 @@
+ * following copyright and licenses apply:
+ *
+ * Copyright 2022 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { FontFaceDefinition } from './src/Application/index.js';
+import { RouterPlatformSettings } from './src/Router/index.js';
+export const Ads: any; // export { default as Ads } from './src/Ads'
+export { AppData, FontFaceDefinition } from './src/Application/index.js';
+export { default as Application } from './src/Application/index.js';
+export const AudioPlayer: any; // export { default as AudioPlayer } from './src/AudioPlayer'
+export const Colors: any; // export { default as Colors } from './src/Colors'
+export { default as Img } from './src/Img/index.js';
+export const Keyboard: any; // export { default as Keyboard } from './src/Keyboard'
+export { default as Launch } from './src/Launch/index.js';
+export { default as Lightning } from './src/Lightning/index.js';
+export const Locale: any; // export { default as Locale } from './src/Locale'
+export const Language: any; // export { default as Language } from './src/Language'
+export const Log: any; // export { default as Log } from './src/Log'
+export const MediaPlayer: any; // export { default as MediaPlayer } from './src/MediaPlayer'
+export const Metrics: any; // export { default as Metrics } from './src/Metrics'
+export const Pin: any; // export { default as Pin } from './src/Pin'
+export const Profile: any; // export { default as Profile } from './src/Profile'
+export const Purchase: any; // export { default as Purchase } from './src/Purchase'
+export const Registry: any; // export { default as Registry } from './src/Registry'
+export { default as Router } from './src/Router/index.js';
+export const Settings: any; // export { default as Settings } from './src/Settings'
+export const Storage: any; // export { default as Storage } from './src/Storage'
+export const TV: any; // export { default as TV } from './src/TV'
+export { default as Utils } from './src/Utils/index.js';
+export const VideoPlayer: any; // export { default as VideoPlayer } from './src/VideoPlayer'
+export const Metadata: any; // export { default as Metadata } from './src/Metadata'
+
+/**
+ * [Platform Settings](https://lightningjs.io/docs/#/lightning-sdk-reference/plugins/settings?id=platform-settings)
+ */
+export interface PlatformSettings {
+  plugins?: {
+    profile?: any;
+    metrics?: any;
+    mediaPlayer?: any;
```

Comment:
the mediaPlayer plugin is deprecated and we really don't recommend it being used anymore (it's also removed from the docs already). Should we maybe not include it here?

Reply:
It's typed as `any` here so we're not really giving the user any guidance on how its used. I can remove it though